nix/pkgs/ts.nix

```diff
@@ -75,46 +75,73 @@
 
   builds = {
     storybook = pkgs.runCommand "storybook" { buildInputs = [ pkgs.yarn ]; } ''
-      tmp=`mktemp -d`
 
+  ############ Create clean ts node package in temp dir
```

Comment:
much better with the comments!

Reply:
Yeah, I think the next step could be, to extract some of these steps to their own functions.

Reply:
or better tackle the root of the problem...